lib/ansible/modules/yum.py

```diff
@@ -241,7 +241,7 @@
         registered to RHN or an RHN Satellite, repoquery allows for querying
         all channels assigned to the system. It is also required to use the
         'list' parameter.
-      - "NOTE: This will run and be logged as a separate yum transation which
+      - "NOTE: This will run and be logged as a separate yum transaction, translation which
```

Comment:
remove `, translation`

Reply:
done

Reply:
i have done the required changes, please look into it @bcoca